PowerEditor/src/MISC/PluginsManager/Notepad_plus_msgs.h

```diff
@@ -36,8 +36,10 @@ enum LangType {L_TEXT, L_PHP , L_C, L_CPP, L_CS, L_OBJC, L_JAVA, L_RC,\
 			   // Don't use L_JS, use L_JAVASCRIPT instead
 			   // The end of enumated language type, so it should be always at the end
 			   L_EXTERNAL};
+enum class LangAutoIndentType { Standard = 1, Extended, Custom };

```

Comment:
Sure. The `Standard = 1` was also one of the constants I left unchecked after finishing writing the code. It was all part of a testing. I reverted back to `0` in my repository.

As for the `Extended` and `Custom`, they work as follow:
- Extended mode will basically integrate the plugin language with Notepad++ C-Style indentation. For instance, the language for my plugin is based on C, so all rules for C-Like indentation apply - hence I don't need to rewrite indentatino code, because you already implemented that.
- As for `Custom` mode - It just tells Notepad++ to NOT process any auto-indentation, because the plugin will be doing it by itself and if Notepad do process indentation, that will cause a conflict with the plugin.